sources/ca/bc/city_of_squamish.json

```diff
@@ -16,17 +16,26 @@
         "addresses": [
             {
                 "name": "city",
-                "data": "https://services.arcgis.com/YCM10gnCAvCoAhpj/arcgis/rest/services/Addressing/FeatureServer/0",
-                "note": "STREET_NUMBER_PREFIX does indeed contain unit info",
-                "protocol": "ESRI",
+                 "data": "https://opendata.arcgis.com/datasets/ae6113adf53540a582defc66dded873a_0.zip?outSR=%7B%22latestWkid%22%3A26910%2C%22wkid%22%3A26910%7D",
```

Comment:
Please don't use download URLs.
Adjust the indenting to match the surrounding file.

Reply:
If you go to the URL in `website`, you can see the URL to use under the APIs dropdown next to download.

Reply:
thank you!